subprojects/docs/src/samples/build-organization/structuring-software-projects/tests/build-android-app.sample.conf

```diff
@@ -1,6 +1,8 @@
 commands: [{
     executable: gradle
-    args: build -x lint -x lintVitalRelease
+    # There's a flaky stacktrace: java.io.FileNotFoundException: https://dl.google.com/android/repository/sys-img/android-desktop/sys-img2-1.xml
+    #  at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1896) ...
```

Comment:
I wonder if we should exclude this stacktrace in our built-in stacktrace checks like we do for other things. Though I suppose for now let's only exclude it for this sample.